imagine-dragons-web/contact.html

```diff
@@ -5,15 +5,17 @@
   <head>
     <script src="https://code.jquery.com/jquery-1.10.2.js"></script>
     <title>The website of Imagine Dragons</title>
```

Comment:
The "tab name" in the browser is very limited and we want the user to immediately know which tab it is. Sometimes, the user can have many tabs and he will see "The website of ...". A better choice would be: "Imagine Dragons - Official Website". 

Also, each page should display the page title.

Reply:
Change this in all pages.